tests/test_011_06_add_into_cart.py

```diff
@@ -0,0 +1,57 @@
+import time
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.service import Service
+
+
+print("\nstart browser...")
+options = webdriver.ChromeOptions()
+options.add_argument("--window-size=1600,1080")
+options.headless = True
+browser = webdriver.Chrome(
+    service=Service(ChromeDriverManager().install()), options=options
+)
+# yield browser
+print("\nquit browser...")
+# browser.quit()
+browser.get("https://www.saucedemo.com/")
+
+
+class Tests:
+    def test_add_items_into_cart(self):
+        browser.find_element(By.ID, "user-name").send_keys("standard_user")
+        browser.find_element(By.XPATH, "//input[@id='password']").send_keys(
+            "secret_sauce"
+        )
+        browser.find_element(By.XPATH, "//input[@name='login-button']").click()
+        browser.get("https://www.saucedemo.com/inventory.html")
+        browser.find_element(
+            By.CSS_SELECTOR, "#add-to-cart-sauce-labs-backpack"
+        ).click()
+        time.sleep(1)
+        browser.find_element(
+            By.CSS_SELECTOR, "#add-to-cart-sauce-labs-bike-light"
+        ).click()
+        time.sleep(1)
```

Comment:
I suggest you don't use `sleep()`, because this will significantly increase the test time unnecessarily and will add additional costs to the testing process